src/plots/Mesh/MeshAttributes.h

```diff
@@ -37,12 +37,14 @@ class MeshAttributes : public AttributeSubject
     enum MeshColor
     {
         Foreground,
-        MeshCustom
+        MeshCustom,
+        MeshRandom
```

Comment:
Have you checked whether old python scripts or saved configurations still work?  Just wanting to see if any coding is needed to process old settings.

Reply:
I haven't tested that. Good suggestion. Will try later this week. This case in particular, the added values to the enum should NOT be a problem except if trying to read *new* session files or scripts into an older VisIt. Are we required to support that too? Or just new VisIt reading old scripts/session files?

Reply:
@biagas can you point me in the right direction for testing this. Where do I start?

Reply:
Ok, in lieu of any further guidance, I ran all current session tests. They all pass. I think this is good to go now.